### PR TITLE
added css for switch component

### PR DESCRIPTION
**Pull request by egmox**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/switch/_switch.scss [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/* ===============Switch================== */
```
Comment (iamsudhanshu) [inline src/components/switch/_switch.scss:1]: Is this theme reviewed by XD?

File: src/components/switch/_switch.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  transition: .4s;
+  border-radius: 34px;
+}
+
+.cmp-adaptiveform-switch__widget-slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
```
Comment (iamsudhanshu) [inline src/components/switch/_switch.scss:45]: Not RTL friendly.

File: src/components/switch/_switch.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+.cmp-adaptiveform-switch__questionmark{
+  display:inline-block;
+  width:1rem;
+  height:1rem;
+  border-radius: 9px;
+  background: url(./resources/images/question.svg) center center / cover no-repeat,#969696;
+  cursor : pointer;
+  // below properties for backward compatibility
+  position: absolute;
+  right: 20px;
```
Comment (iamsudhanshu) [inline src/components/switch/_switch.scss:90]: @egmox Have you validated with RTL?

Comment (egmox) [reply src/components/switch/_switch.scss:86]: @iamsudhanshu RTL is validated. There is a class for rtl at line #89

File: src/components/switch/_switch.scss (hunk 1 above)
Comment (egmox) [reply src/components/switch/_switch.scss:1]: Yes, comments given by XD was incorporated in the theme

File: src/components/switch/_switch.scss [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+.cmp-adaptiveform-switch__widget {
+  opacity: 0;
+  width: 0;
+  height: 0;
+  display: block;
+}
+
+.cmp-adaptiveform-switch__widget-slider {
+  cursor: pointer;
+  background-color: #ccc;
```
Comment (amit-mnnit) [inline src/components/switch/_switch.scss:23]: use variable for color

File: src/components/switch/_switch.scss [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  border-radius: 34px;
+  transition: all .4s;
+  width: 60px;
+  height: 32px;
+  position: relative;
+}
+
+input:checked + .cmp-adaptiveform-switch__widget-slider {
+  background-color: #2196F3;
```
Comment (amit-mnnit) [inline src/components/switch/_switch.scss:32]: use variable for color

File: src/components/switch/_switch.scss [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+input:focus + .cmp-adaptiveform-switch__widget-slider {
+  box-shadow: 0 0 1px #2196F3;
```
Comment (amit-mnnit) [inline src/components/switch/_switch.scss:36]: use variable for color

File: src/components/switch/_switch.scss [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+.cmp-adaptiveform-switch__circle-indicator{
+  height: 24px;
+  width: 24px;
+  background-color: #fff;
```
Comment (amit-mnnit) [inline src/components/switch/_switch.scss:42]: use variable for color

File: src/components/switch/_switch.scss [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+[dir="rtl"] .cmp-adaptiveform-switch__help-container .cmp-adaptiveform-switch__questionmark{
+  position: unset;
+  right: unset;
+  left: unset;
+}
+.cmp-adaptiveform-switch__shortdescription {
+  font-size: $font-sm;
+  margin-top: $error-top-margin;
+}
+.cmp-adaptiveform-switch[data-cmp-valid=false]{
```
Comment (amit-mnnit) [inline src/components/switch/_switch.scss:113]: please update this class like this https://github.com/adobe/aem-forms-theme-canvas/blob/main/src/components/textinput/_textinput.scss#L63
It is not working for RTl

same for this cmp-adaptiveform-switch[data-cmp-valid=true] class

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
Please check commnets

**Review by amit-mnnit — APPROVED** [review]

**Pull request merged**